### ISSUE TITLE
GenomicsDB does not support non-diploid data

### ISSUE THREAD

**Issue by droazen** [broadinstitute/gatk#3342]
As discussed in https://github.com/broadinstitute/dsde-docs/issues/2303, `GenomicsDB` does not yet support non-diploid data. We should remedy this!

**Comment by droazen** [comment]
For @kgururaj and @kdatta

**Comment by vdauwera** [comment]
@kgururaj and @kdatta Here's some GVCF data for a trio of samples, each called to haploid (ploidy 1) and tetraploid (4) genotypes. I included the reference (just chromosome 20) and the intervals list. This is data from one of our workshop tutorials so many of the intervals in the list I used don't have any data (so lots of no-calls) but there should be enough usable calls for testing purposes. Let me know if this isn't sufficient to get you started. 

Thanks for looking into this, it's very important for a non-trivial subset of our users. 

[genomicsdb.zip](https://github.com/broadinstitute/gatk/files/1171416/genomicsdb.zip)
